news/2020-03-20-beatmap-spotlights-winter-2020.md

```diff
@@ -0,0 +1,239 @@
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/lights.jpg)](https://osu.ppy.sh/beatmapsets/1044462)
+
+[Kukkai](https://osu.ppy.sh/users/7811952)'s "Lights of Muse" has a certain feeling behind it that is fairly uncommon these days, reminiscent of something that would have easily had a play count in the hundred thousand if it was ranked a few years ago. While the use of patterns and rhythm is fairly simple, the whole set is at a consistent level of fun that is hard to come by.
+
+*written by [Sinnoh](https://osu.ppy.sh/users/4236057)*
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/catch/sweet.jpg)](https://osu.ppy.sh/beatmapsets/738169)
+
+[Sorcerer](https://osu.ppy.sh/users/1913190) and [ZiRoX](https://osu.ppy.sh/users/200768) has something sweet to offer in this season's Spotlights!
+
+Starting from the bottom we have Cup, Salad and Platter all made by Sorcerer, and they lead the player through the learning stages of the streams and how to control the jumps. If you can already get full combo on the Salad then the next step is to learn how to handle the Hyperjumps and time your normal jumps at the end of the streams in Platter. Very well designed difficulty curve.
+
+Next up we have two Rain to choose from, one from the mapper of the previous three and one guest difficulty from ZiRoX. Dispite their similar star ratings there is a clear difference in them. ZiRoX decided to use less Hyperjump during the stream parts but instead use stair patterns, which are not necessarily easier to handle though. Meanwhile Sorcerer decided to step up from Platter and make his Hyperjump streams even more deadly for the player with more experience.
+
+And for last we have a Sweet Overdose, which is not really that sweet if you take a peek inside. It raises to jump stream pattern into a whole new level where the players have to face the true evil in case they dare to challenge it. The sweetness flew away and only Rain remained, raining fruits and Hypers everywhere. Prepare your umbrella and get ready to catch 'em all!
+
+*written by [Kurokami](https://osu.ppy.sh/users/260933)*
+
+## <a id="mania"></a> osu!mania
+
+[![](/wiki/shared/news/2020-03-20-beatmap-spotlights-winter-2020/mania/transhuman.jpg)](https://osu.ppy.sh/beatmapsets/1069426)
+
+Incredibly exciting yet surprisingly delicate in its structure, [Kamuy](https://osu.ppy.sh/users/7439226)'s Transhumanist stands out among currently ranked maps by being the first semi-dump ranked in osu!mania. This map has been making waves in both mapping as well as higher-level playing communities by combining consistent usage of pattern/snap arrangements to represent specific kinds of wubs in the song in order to fully represent intensity with player-friendly yet challenging patterns. These make it an absolute joy to play through and fascinating to pick apart in the editor as well.
```

Comment:
what does semi-dump mean exactly?

Reply:
It's a rather ambiguous term? Dumps are usually described as placing notes where there isn't a distinct - yet continuous - sound, but it still flows with the music; it typically uses certain snaps to represent the sound(s). Because of this, it is one of the reasons why it can be hard to describe what a dump exactly is. Semi-dump is pretty much where you only do this for specific sounds (for example in this map, the dump is exclusive to the wubs in the song).